apps/condo/domains/ticket/schema/Ticket.test.js

```diff
@@ -3371,8 +3373,14 @@ describe('Ticket', () => {
         })
 
         describe('Ticket created', () => {
+            let organization
+
             beforeAll(async () => {
+                const [testOrganization] = await createTestOrganization(admin)
+                organization = testOrganization
```

Comment:
you can do it like this `[organization] = await createTestOrganization(admin)`
if he swears at the next line, then you can do this `[organization] = await createTestOrganization(admin);`
does not affect the approval, just wrote that there is a shorter option